configuration/config_ESPO-G_RDRS.yml

```diff
@@ -110,7 +110,7 @@ extraction:
             match_hist_and_fut: True
             allow_conversion: True
             allow_resampling: False
-            restrict_members:
+            restrict_members: # Does restrict_members does the same thing as member in other_search_criteria ?
```

Comment:
member in `other_search_criteria` you have to name the member you want ( eg. r1i1p1f1). This means take the first one available.